src/frontend/utils/updateLiveParticipants.ts

```diff
@@ -0,0 +1,79 @@
+import { updateLiveParticipants } from 'data/sideEffects/updateLiveParticipants';
```

Comment:
Shouldn't we handle the state only in the backend to prevent concurrency issue ?
I mean, shouldn't we have an endpoint to add a participant, to remove one etc and not just an endpoint to update the current state ?

Reply:
I'm not sure to get your point here.
How would concurrency happen ?

Reply:
Any user with permission `can_update` will trigger updates to the state store in the backend.
Since I don't know if we are sure there is only one user with this kind of permission, I will make the assumption we can have multiple users with this permission.

So I one user accept somone on stage when at the same time an other one kick someone from stage your backend state will be "corrupted".

Reply:
Having multiple instructors at the same time can happen and this is one of the reason we need a persisted state.